src/features/components/templates/signup/with-banner/social/index.tsx

```diff
@@ -11,6 +12,7 @@ import GoogleLogo from 'images/svg/google_logo.svg'
 import { TString } from 'types/generics'
 import Login, { TSocialProvider } from 'common/login'
 import { IconType } from 'features/types'
+import { isBrowser } from 'common/utility'
```

Comment:
Seems like this import is redundant, could u pls remove it 🙏

Reply:
sure